Update Readme.md (#8)
DevExpress-Examples/how-to-display-a-line-when-the-dxgrid-doesnt-include-any-record-e1786: Readme.md

[![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E1786)
[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
<!-- default badges end -->

> # WPF Data Grid - Display Text if the Grid Contains no Records

Reviewer: ```suggestion
# WPF Data Grid - How to Display Text in a Grid With No Records
```
